### PR TITLE
Add Forester theme

### PR DESCRIPTION
**Pull request by ivryakhov**
A new theme have been added

### CODE REVIEW

File: VSColorThemes/Themes/Forester.xml [hunk 1]
```diff
@@ -0,0 +1,8400 @@
...
+      <Color Name="LiveMetricsFailedRequestFill">
+        <Background Type="CT_RAW" Source="FF4C2728" />
+      </Color>
+      <Color Name="SquigglyOrange">
+        <Background Type="CT_RAW" Source="00FF9427" />
+      </Color>
+    </Category>
+    <Category Name="Autos" GUID="{a7ee6bee-d0aa-4b2f-ad9d-748276a725f6}">
+      <Color Name="ChangedText">
+        <Background Type="CT_INVALID" Source="00000000" />
```
Comment (Newrad0603) [inline VSColorThemes/Themes/Forester.xml]: Please remove all instances of CT_INVALID color entries. They are non-functional, but bloat the private registry.

File: VSColorThemes/Themes/Forester.xml [hunk 2]
```diff
@@ -0,0 +1,8400 @@
...
+        <Foreground Type="CT_RAW" Source="FFF7A1A1" />
+      </Color>
+      <Color Name="Plain Text">
```
Comment (Newrad0603) [inline VSColorThemes/Themes/Forester.xml]: Once the CT_INVALID entries are removed, you'll end up with empty colors, so make sure to remove those as well. Once the empty colors are removed, you may have empty categories which should also be removed.

**Comment by Newrad0603** [comment]
Can you please provide an overview screenshot of the theme?

**Comment by ivryakhov** [comment]
forester_theme_sample
this is a sample screenshot

**Comment by ivryakhov** [comment]
[Forester.zip](https://github.com/microsoft/VS-ColorThemes/files/3359119/Forester.zip)
The theme file to import to the extension if someone interested in and eager to try this.

**Pull request merged**